solutions/instances/main.tf

```diff
@@ -245,7 +250,7 @@ module "observability_instance" {
       skip_cos_auth_policy = var.ibmcloud_cos_api_key != null ? true : var.skip_cloud_logs_cos_auth_policy
     },
     metrics_data = {
-      enabled              = true
+      enabled              = false # Cloud logs provision fails if metrics COS bucket is attached - https://github.ibm.com/GoldenEye/issues/issues/11418#issuecomment-96873794
```

Comment:
We should not do this in the DA. It will remove COS bucket for people who already have them attached. We need to wait for the fix